book/src/commands.md

```diff
@@ -0,0 +1,25 @@
+# Commands
+
+Commands in Halloy are prefixed with `/`.
+
+Example
+
+```
+/me says halloy!
+```
+
+Halloy will first try to run below commands, and lastly send it directly to the server.
+
+| Command | Alias      | Description                                                   |
+|---------|------------|---------------------------------------------------------------|
+| `away`  |            | Mark yourself as away. If already away, the status is removed |
+| `join`  | `j`        | Join channel(s) with optional key(s)                          |
+| `me`    | `describe` | Send an action message to the channel                         |
+| `mode`  | `m`        | Set mode(s) on a channel or retrieve the current mode(s) set  |
+| `msg`   | `query`    | Open a query with a nickname and send an optional message     |
```

Comment:
This description doesn't seem entirely accurate, since `query` is only an alias of `/msg` and `msg` targets can be channels or nicknames